packages/world/ts/concatBaseAbi.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, expect, it } from "vitest";
+import { Abi } from "viem";
+import worldAbi from "./mocks/worldAbi";
+import { concatBaseAbi } from "./concatBaseAbi";
+import combinedAbi from "./mocks/combinedAbi";
+
+describe("World ABI", () => {
+  it("should concat base and world ABI", () => {
+    const abi = concatBaseAbi(worldAbi as Abi);
+    expect(abi).toMatchObject(combinedAbi);
```

Comment:
tests are meant to describe and demonstrate behavior, but that's really hard to see when these objects are huge and hidden away in files

can we do some smaller tests instead of using mocks? like

```
const abi = concatBaseAbi([{type: "function", name: "addTask", ...}]);
expect(abi).toContainEqual({ type: "function", name: "addTask", ... });
```

Reply:
agreed, I made it so verbose to make sure it works with "real-world" data, and also handles correctly events deduplication, etc. and then also knew what output to expect. But regardless, it could have been split into smaller pieces.

Anyhow, now it's clear that since the fetched world ABI only contains functions, the testing can be simplified, updated to your suggestion.